AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomMaterialEditor_BasicTests.py

```diff
@@ -68,139 +69,139 @@ def run():
     """
 
     # 1) Test Case: Opening an Existing Asset
-    document_id = material_editor.open_material(MATERIAL_TYPE_PATH)
-    print(f"Material opened: {material_editor.is_document_open(document_id)}")
+    document_id = atom_tools_utils.open_material(MATERIAL_TYPE_PATH)
+    print(f"Material opened: {atom_tools_utils.is_document_open(document_id)}")
 
     # Verify if the test material exists initially
     target_path = os.path.join(azlmbr.paths.projectroot, "Materials", NEW_MATERIAL)
     print(f"Test asset doesn't exist initially: {not os.path.exists(target_path)}")
 
     # 2) Test Case: Creating a New Material Using Existing One
-    material_editor.save_document_as_child(document_id, target_path)
-    material_editor.wait_for_condition(lambda: os.path.exists(target_path), 2.0)
+    atom_tools_utils.save_document_as_child(document_id, target_path)
+    atom_tools_utils.wait_for_condition(lambda: os.path.exists(target_path), 2.0)
     print(f"New asset created: {os.path.exists(target_path)}")
     time.sleep(2.0)
```

Comment:
wait_for_condition will already help synchronize, are these blind sleeps necessary?

Reply:
this test is getting updated in a future internal ticket, so disregard those specifics in this PR - it just needed to be updated because of the renames

Reply:
The sleeps were from the time when material editor relied on the asset processor for opening and saving documents. That dependency was part of the reason the tests were flaky and eventually sandboxed.